src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -3119,11 +3124,100 @@ def generate(cls, config: SubsetConfig) -> Compose:
 
         transforms = []
         for cfg_transform in config.transforms:
+            if (input_size := getattr(config, "input_size", None)) is not None:
+                cls._eval_input_size(cfg_transform, input_size)
             transform = cls._dispatch_transform(cfg_transform)
             transforms.append(transform)
 
         return Compose(transforms)
 
+    @classmethod
+    def _eval_input_size(cls, cfg_transform: dict[str, Any], input_size: int | tuple[int, int] | None) -> None:
+        """Evaluate the input_size and replace the placeholder in the init_args.
+
+        Input size should be specified as $(input_size). (e.g. $(input_size) * 0.5)
+        Only simple multiplication or division evaluation is supported. For example,
+        $(input_size) * -0.5    => supported
+        $(input_size) * 2.1 / 3 => supported
+        $(input_size) + 1       => not supported
+        The function decides to pass tuple type or int type based on the type hint of the argument.
+        float point values are rounded to int.
+        """
+        if input_size is None:
+            return
```

Comment:
It seems this condition is not passed because checking whether `input_size` is None is already in L3127.
I think we can use it and remove L3127.

Reply:
I think leaving the safeguard is better choice. How leaving it as it is?

Reply:
If so, let's keep this.